### PR TITLE
Autocomplete on asset tags + autocomplete style

### PR DESCRIPTION
**Pull request by arvida**
This makes the autocompletion of tags work. Plus it gets some style.

Ping @joeljunstrom 

Trello card:
https://trello.com/c/VAozS8UV

### CODE REVIEW

File: app/assets/javascripts/admin/assets.js [hunk 1]
```diff
@@ -92,26 +92,30 @@
...
-      $('#asset_tag_names').bind({
-        keyIn: function(ev) {
-          var tag_list = ev.smartAutocompleteData.query.split(" ");
-          ev.smartAutocompleteData.query = $.trim(tag_list[tag_list.length - 1]);
-        },
-        itemSelect: function(ev, selected_item) {
-          var options = $(this).smartAutoComplete();
-          var selected_value = $(selected_item).text();
-          var cur_list = $(this).val().split(" ");
+      $('.asset_tags').each(function(_, el){
```
Comment (joeljunstrom) [inline app/assets/javascripts/admin/assets.js]: Do we really need to iterate here?

Would'nt 

```
var $tags = $('.asset_tags');
$tags.smartAutoComplete({ // options });
$tags.bind({ // options });
```

do the same thing? (or even just chaining smartAutocomplete and bind

Also, awesome if we can find something else than just a class to do the lookup on (ie. $("input.asset_tags") or $("#asset_form").find("input.asset_tags");

Comment (arvida) [inline app/assets/javascripts/admin/assets.js]: I didn't get it to work without iterating over the elements. It just got bind to the last element when I didnt add it for each element. Any ideas?

**Comment by arvida** [comment]
Force pushed better for selector for triggering autocomplete functionallity.

File: app/assets/javascripts/admin/assets.js (hunk 1 above)
Comment (joeljunstrom) [inline app/assets/javascripts/admin/assets.js]: Cant see your comment on the page but might be a bad plugin then. It's not important.

**Pull request merged**